assets/css/style.css

```diff
@@ -0,0 +1,58 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
```

Comment:
You can take out this body styling here.  I don't see it making a difference on the page.

Reply:
box-sizing: border-box; is there to avoid scroll left or right